dell/xps/15-9550/nvidia/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, pkgs, ... }:
```

Comment:
Please also reference this file in the README table as well as flake.nix.

Reply:
Done.